v3/src/components/graph/adornments/count/count-adornment-model.ts

```diff
@@ -37,11 +37,14 @@ export const CountAdornmentModel = AdornmentModel
       // the divisor)
       const categoricalAttrCount = dataConfig?.categoricalAttrCount ?? 0
       const hasPercentTypeOptions = categoricalAttrCount > 1
+      const rowCases = dataConfig?.filteredForShowMeasuresForSelection(dataConfig?.rowCases(cellKey) ?? []) ?? []
+      const columnCases = dataConfig?.filteredForShowMeasuresForSelection(dataConfig?.columnCases(cellKey) ?? []) ?? []
+      const cellCases = dataConfig?.filteredForShowMeasuresForSelection(dataConfig?.cellCases(cellKey) ?? []) ?? []
       const divisor = hasPercentTypeOptions && self.percentType === "row"
-        ? dataConfig?.rowCases(cellKey).length ?? 0
+        ? rowCases.length ?? 0
         : hasPercentTypeOptions && self.percentType === "column"
-          ? dataConfig?.columnCases(cellKey).length ?? 0
-          : dataConfig?.cellCases(cellKey).length ?? 0
+          ? columnCases.length ?? 0
+          : cellCases.length ?? 0
```

Comment:
You've guaranteed that `rowCases`, `columnCases`, and `cellCases` are always defined above, so you shouldn't need the `?? 0` here.